_data/projects/SunnySunday.yml

```
@@ -0,0 +1,13 @@
+name: SunnySunday
+desc: A weather web app using Google Maps and Openweathermap
+site: https://github.com/bacloud14/SunnySunday
+tags:
+  - weather
+  - google-maps
+  - nodejs
```

:art: to address the linter feedback

```suggestion
  - node.js
```